packages/stream_chat_flutter/lib/src/message_widget.dart

```diff
@@ -837,6 +840,22 @@ class _MessageWidgetState extends State<MessageWidget>
         [];
 
     if (images.length > 1) {
```

Comment:
Now that we are passing `List<Attachment>` instead of a single `Attachment` to the `AttachmentWidget`, I would suggest we move the attachment `grouping logic` to their respective widget instead of handling it in `MessageWidget`.